### PR TITLE
fix: replace antd select with hew select

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
[WEB-1820]

## Test Plan
- Check if there's no place antd select component is used
- Check if all replaced select components has solid background
- Check if all replaced select components work as expected

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

[WEB-1820]: https://hpe-aiatscale.atlassian.net/browse/WEB-1820?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by keita-determined** [comment]
@julian-determined-ai fixed the width issue

**Review by julian-determined-ai — APPROVED** [review]
LGTM!

File: webui/react/src/pages/Cluster/ClusterHistoricalUsageCsvModal.tsx [hunk 1]
```diff
@@ -1,14 +1,13 @@
-import { Form, Modal, Select } from 'antd';
+import { Form, Modal } from 'antd';
 import dayjs, { Dayjs } from 'dayjs';
 import DatePicker from 'hew/DatePicker';
-import Icon from 'hew/Icon';
+// import Icon from 'hew/Icon';
```
Comment (julian-determined-ai) [inline webui/react/src/pages/Cluster/ClusterHistoricalUsageCsvModal.tsx:4]: nit: Remove this commented out line

Comment (keita-determined) [reply webui/react/src/pages/Cluster/ClusterHistoricalUsageCsvModal.tsx]: done

**Pull request merged**